Dshop/apps/products_catalogue/views.py

```diff
@@ -1,3 +1,15 @@
-#from django.shortcuts import render
+from django.shortcuts import render
+from django.views import View
+from .models import Product
 
 # Create your views here.
+class ProductListView(View):
```

Comment:
Please change to ListView
https://akademiait.com.pl/class-based-views-w-django-wygodny-sposob-tworzenia-widokow-czesc-2/

Reply:
fixed :)